subcommand/connect-init/command.go

```diff
@@ -24,7 +24,7 @@ const (
 	// The number of times to attempt ACL Login.
 	numLoginRetries = 3
 	// The number of times to attempt to read this service (60s).
-	defaultServicePollingRetries = 60
+	defaultServicePollingRetries = 120
```

Comment:
this change seems unrelated?

Reply:
yeah..i just crammed it into this PR but I can remove it from this one and add another PR for the same